### PR TITLE
Fix/issues found in testing

### PR DESCRIPTION
**Pull request by wojtekzyla**
- Add warnings when deleting groups or profiles which are configured in the inventory. 
- No frequency configuration for walk profiles. 
- Default number of presented items changed to 20.
- Save configuration from mongo to values.yaml on host machine after clicking 'Apply change'

### CODE REVIEW

File: backend/SC4SNMP_UI_backend/apply_changes/config_to_yaml_utils.py [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+    @abstractmethod
+    def convert(self, documents: list) -> dict:
+        pass
+
+
+class ProfilesToYamlDictConversion(MongoToYamlDictConversion):
+    def convert(self, documents: list) -> dict:
+        """
+        ProfilesToYamlDictConversion converts profiles from mongo collection to
+         format that can be dumped to yaml file
```
Comment (omrozowicz-splunk) [inline backend/SC4SNMP_UI_backend/apply_changes/config_to_yaml_utils.py]: extra space here

File: backend/SC4SNMP_UI_backend/apply_changes/config_to_yaml_utils.py [hunk 2]
```diff
@@ -0,0 +1,224 @@
...
+            result[profile_name]["varBinds"] = var_binds
+
+        return result
+
+
+class GroupsToYamlDictConversion(MongoToYamlDictConversion):
+    def convert(self, documents: list) -> dict:
+        """
+        GroupsToYamlDictConversion converts groups from mongo collection to
+         format that can be dumped to yaml file
```
Comment (omrozowicz-splunk) [inline backend/SC4SNMP_UI_backend/apply_changes/config_to_yaml_utils.py]: and here

File: backend/SC4SNMP_UI_backend/apply_changes/config_to_yaml_utils.py [hunk 3]
```diff
@@ -0,0 +1,224 @@
...
+
+
+class MongoToYamlDictConversion:
+    """
+    MongoToYamlDictConversion is an abstract class. Implementations of this class converts
+    appropriate mongo collections to dictionaries in such a way, that configurations from those collections can be
+    dumped to yaml file with appropriate formatting.
+    """
+    @classmethod
+    def yaml_escape_list(cls, *l):
```
Comment (omrozowicz-splunk) [inline backend/SC4SNMP_UI_backend/apply_changes/config_to_yaml_utils.py:25]: can you add comment with description of what this method do generally?

File: backend/SC4SNMP_UI_backend/apply_changes/config_to_yaml_utils.py [hunk 4]
```diff
@@ -0,0 +1,224 @@
...
+        :param document: dictionary with profiles configuration
+        :param delete_tmp: whether to delete temporary file after parsing
+        :return: parsed configuration ready to be saved to values.yaml
+        """
+        self._save_temp(document)
+        lines = ""
+        with open(self._file_path, "r") as file:
+            line = file.readline()
+            while line != "":
+                lines += f"{line}"
```
Comment (omrozowicz-splunk) [inline backend/SC4SNMP_UI_backend/apply_changes/config_to_yaml_utils.py]: wouldn't `lines += line` be enough?

**Review by omrozowicz-splunk — CHANGES_REQUESTED** [review]

**Review by ajasnosz — APPROVED** [review]

File: backend/SC4SNMP_UI_backend/apply_changes/config_to_yaml_utils.py (hunk 4 above)
Comment (wojtekzyla) [reply backend/SC4SNMP_UI_backend/apply_changes/config_to_yaml_utils.py]: Done

File: backend/SC4SNMP_UI_backend/apply_changes/config_to_yaml_utils.py (hunk 3 above)
Comment (wojtekzyla) [reply backend/SC4SNMP_UI_backend/apply_changes/config_to_yaml_utils.py:25]: Done

File: backend/SC4SNMP_UI_backend/apply_changes/config_to_yaml_utils.py (hunk 2 above)
Comment (wojtekzyla) [reply backend/SC4SNMP_UI_backend/apply_changes/config_to_yaml_utils.py]: Done

File: backend/SC4SNMP_UI_backend/apply_changes/config_to_yaml_utils.py (hunk 1 above)
Comment (wojtekzyla) [reply backend/SC4SNMP_UI_backend/apply_changes/config_to_yaml_utils.py]: Done

**Review by omrozowicz-splunk — APPROVED** [review]

**Pull request merged**